mldsa/fips202/fips202.c

```diff
@@ -475,7 +475,7 @@ __contract__(
  *              Modifies the state. Can be called multiple times to keep
  *              squeezing, i.e., is incremental.
  *
- * Arguments:   - uint8_t *out: pointer to output
+ * Arguments:   - uint8_t *out: pointer to mutable output data
  *              - size_t outlen: number of bytes to be squeezed (written to out)
```

Comment:
Can you use add ` /* Reference: ... */` comment while at it, noting the change in the implementation?

Reply:
DOne